FCC Survey Page/css/sp-css.css

```diff
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans+JP|Roboto+Slab&display=swap');
+
+html,
+body {
+    background-color: #A9D7D1;
+}
+
+#form-outer {
+    border: 5px;
+    border-radius: 10px;
+    max-width: 65%;
+    margin: 0 auto;
+    background-color: #FAFAFA;
+    padding: 0.5em;
+}
+
+#title {
+    font-family: 'Noto Sans JP', sans-serif;
+    font-size: 2em;
+    text-align: center;
+ }
+
+#description {
+    margin: 1.2em 0 1.2em 0;
```

Comment:
**Recommended:** Set the margin to 20px from all sides. The space would look consistent from all sides. Better from designer point of view. It's important to have consistency through the site.